src/sql/azdata.proposed.d.ts

```diff
@@ -38,6 +38,10 @@ declare module 'azdata' {
 			nbKernelAlias?: string
 		}
 
+		export interface ICellOutput {
+			id?: string;
```

Comment:
These would just be comments about VSCode compatibility, which we've already decided are unnecessary. Do you still want me to add something here? These are extensions to existing interfaces.

Reply:
I would at least add what this "id" is (even if it's just a generic description) and what it could be used for.

But also - is this actually necessary to have in the public API? If users aren't expected to know/care about this then we should likely instead be putting this in internal definitions instead and not here.

Reply:
We only pass around things as this interface from what I can tell, so if we wanted to define it at all it would need to be here.

Reply:
That's just by convention - and one that would be great to fix one of these days (similar to how VS Code has stuff like the DTOs for transferring stuff around internally). The fact that we use azdata types in core code is not great and we should avoid doing it where possible. But for now it's fine - but we should still always be adding doc comments on any of the public APIs so people know what to expect when using it.